apps/studio/tests/vitestSetup.ts

```diff
@@ -34,10 +35,6 @@ beforeAll(() => {
       }),
     }
   })
-
-  vi.mock('next/compat/router', () => require('next-router-mock'))
-
-  routerMock.useParser(createDynamicRouteParser(['/projects/[ref]']))
```

Comment:
want to make sure that this isn't accidentally resetting the list, so I'm removing it as this should be covered in the glob utility anyways